core/services/functions/listener.go

```diff
@@ -348,7 +362,37 @@ func (l *FunctionsListener) handleRequest(ctx context.Context, requestID [32]byt
 		return
 	}
 
-	computationResult, computationError, domains, err := eaClient.RunComputation(ctx, requestIDStr, l.job.Name.ValueOrZero(), subscriptionOwner.Hex(), subscriptionId, "", requestData)
+	nodeProvidedSecrets := ""
+	if l.decryptor != nil && requestData.SecretsLocation == 1 && len(requestData.Secrets) > 0 {
```

Comment:
There should be an enum in gethwrappers instead of "1"

Reply:
Looks like there isn't...

Reply:
I see you added this const value in your PR.  Will rebase & use that value once your PR is merged.